### PR TITLE
Add new glossary.md file.

### PR DESCRIPTION
**Pull request by janet-can**

### CODE REVIEW

**Comment by janet-can** [comment]
@vijaykiran I'm a little worried-- what happened to all the checks that are supposed to be performed on a new PR? I see only one check that passed and that is not what I expected. Did I miss something??

**Comment by dirkgroenen** [comment]
> @vijaykiran I'm a little worried-- what happened to all the checks that are supposed to be performed on a new PR? I see only one check that passed and that is not what I expected. Did I miss something??

GitHub is experiencing some issues which is causing those tests to stall. It's not you! 🙂 https://www.githubstatus.com/

**Comment by janet-can** [comment]
> > @vijaykiran I'm a little worried-- what happened to all the checks that are supposed to be performed on a new PR? I see only one check that passed and that is not what I expected. Did I miss something??
> 
> GitHub is experiencing some issues which is causing those tests to stall. It's not you! 🙂 https://www.githubstatus.com/

WHEW! Holy smokes, I thought I was going crazy. Thank you. I will just wait it out. Back to sleep, @dirkgroenen !

File: docs/documentation/glossary.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+### scan
+A Soda SQL CLI command that uses SQL queries to extract information about data in a database table.
+
+### scan YAML
+The file in which you configure scan metrics and tests. Soda SQL uses the input from this file to prepare, then run SQL queries against your data. See [Scan YAML]({% link documentation/scan.md %}).
+
+### SQL metric
+A custom metric you define in your Scan YAML file. SQL metrics essentially enable you to add SQL queries to your Scan YAML file so that Soda SQL runs them during a scan. See [SQL metrics]({% link documentation/sql_metrics.md %}#sql-metrics).
+
+### table
```
Comment (vijaykiran) [inline docs/documentation/glossary.md:46]: Maybe just say a “table in your data warehouse “ ? Same for the column “a column in your warehouse tables”

File: docs/documentation/glossary.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+### column metric
+A property of the data of a single column in your database. Use a column metric to define tests that apply to specific columns in a table during a scan. See [Column metrics]({% link documentation/sql_metrics.md %}#column-metrics).
+
+### configuration key
+The key in the key-value pair that you use to define configuration in your Scan YAML file. See [Scan YAML configuration keys]({% link documentation/scan.md %}#scan-yaml-configuration-keys).
+
+### create
+A Soda SQL CLI command that creates a warehouse directory.
+
+### default metrics
```
Comment (vijaykiran) [inline docs/documentation/glossary.md:28]: metrics -> metric (just to keep consistency in glossary)

File: docs/documentation/glossary.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+### scan YAML
+The file in which you configure scan metrics and tests. Soda SQL uses the input from this file to prepare, then run SQL queries against your data. See [Scan YAML]({% link documentation/scan.md %}).
+
+### SQL metric
+A custom metric you define in your Scan YAML file. SQL metrics essentially enable you to add SQL queries to your Scan YAML file so that Soda SQL runs them during a scan. See [SQL metrics]({% link documentation/sql_metrics.md %}#sql-metrics).
+
+### table
+A SQL table.
+
+### table metrics
```
Comment (vijaykiran) [inline docs/documentation/glossary.md:49]: metrics -> metric (just to keep consistency in glossary)

**Pull request merged**